components/hab/tests/test_install_script.ps1

```diff
@@ -0,0 +1,22 @@
+Describe "Install habitat using install.ps1" {
+    It "can install the latest version of Habitat" {
+        components/hab/install.ps1 -c dev
```

Comment:
maybe a better check here than just checking exit code would be to ensure that `(get-command hab).Path` equals `C:\ProgramData\Habitat\hab.exe` which is where install.ps1 installs to.